pkg/apiserver/webhook/cloudentityselector_webhook.go

```diff
@@ -48,10 +46,11 @@ var (
 		"use vpc matchID instead of vpc matchName"
 	errorMsgMatchIDNameTogether = "matchID and matchName are not supported together, " +
 		"configure either matchID or matchName in an EntityMatch"
-	errorMsgAccountNameUpdate    = "account name update not allowed"
-	errorMsgSameAccountUsage     = "account in a namespace can be owner of only one CloudEntitySelector"
-	errorMsgOwnerAccountNotFound = "failed to find owner account"
-	errorMsgInvalidCloudType     = "invalid cloud provider type"
+	errorMsgAccountNameUpdate         = "account name update not allowed"
```

Comment:
reminder: Change name to Name and namespace to Namespace.

Reply:
I don't think it matters, there was a comment earlier to change from uppercase to lowercase. Thats when I did it.